bundle/language_en.properties

```diff
@@ -1197,6 +1197,7 @@ angal.newbill.payment.btn.key
 angal.newbill.paymentmadebeforebilldate.msg                                                            = Payment made before bill date.
 angal.newbill.paymentreceipt.btn                                                                       = Payment Receipt
 angal.newbill.paymentreceipt.btn.key                                                                   = N
+angal.newbill.pleasefindpatientbeforeremotehim.msg                                                     = Please find the patient first before trying to remove him.
```

Comment:
I think "him"/"her" is not very professional here, also I don't think the suggestion make sense (no one wants to find a record in order to suddenly remove it).

Why we don't make just the button doing nothing if patient is null?
Also, we can disable the button when no patient is selected

Reply:
I remove that message bundle